livekit_models.proto

```diff
@@ -169,3 +169,49 @@ message ClientInfo {
   string browser_version = 8;
   string address = 9;
 }
+
+// server provided client configuration
+message RTCClientConfiguration {
+  AudioConfiguration audio = 1;
+  VideoConfiguration video = 2;
+  VideoConfiguration screen = 3;
+  SignalConfiguration signal = 4;
+}
+
+message AudioConfiguration {
+  message AudioProcessing {
```

Comment:
how should we interpret the values in this message? does it mean

* client should use <feature> by default
* client supports/does not support <feature>

when a user requests for these features explicitly, how would we reconcile the final decision?